README.md

```diff
@@ -14,6 +14,16 @@ This tells `black` how long each line of Python code can be.
 
 **Default**: `81`, which is _different_ from `black`'s default of `88`.
 
+## Include (optional)
+This tells `black` which files we should verify the format.
+
+**Default**: `\.pyi?$`, which is `black`'s default value if not informed.
+
+## Exclude (optional)
+This tells `black` which files we should ignore.
+
+**Default**: `(\.direnv|\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|\.svn|_build|buck-out|build|dist)`, which is `black`'s default value if not informed.
```

Comment:
Doesn't match the default value [in the black repository](https://github.com/psf/black#command-line-options), as it's missing the leading and trailing `/`.

Reply:
Weird ... tried yesterday with the trailling and ending slash, but did not worked. That's why I removed.
Anyway, tried again now and it seems to be working. Have a look please